.github/workflows/pr-loadtest.yml

```diff
@@ -0,0 +1,65 @@
+name: PR Load Test
+
+# Trigger the test Manually additionally provide PR number.
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'Pull Request Number'
+        required: true
+
+env:
+  IMG: europe-docker.pkg.dev/kyma-project/dev/telemetry-manager:PR-${{ github.event.inputs.pr_number }}
+  K8S_VERSION: "1.27"
+
+jobs:
+  load-test:
+    strategy:
+      max-parallel: 3
+      matrix:
+        argument: ["-n pr-lt-traces -t traces", "-n pr-lt-traces-m -t traces -m true", "-n pr-lt-traces-b -t traces -b true", "-n pr-lt-metrics -t metrics", "-n pr-lt-metrics-m -t metrics -m true", "-n pr-lt-metrics-b -t metrics -b true"]
```

Comment:
I cannot find logs here

Isn't the pattern always the same? So can't we define two arrays out of that permutating the options?
Can we improve the script to also support long argument names or do you remember what -b will do?